products/sle/main.pm

```diff
@@ -1024,6 +1024,22 @@ else {
         loadtest 'qa_automation/patch_and_reboot' if is_updates_tests;
         loadtest 'network/vsftpd';
     }
+    elsif (get_var('QAM_AUTOFS')) {
```

Comment:
why only on SLES? it cant work on openSUSE too ?

Reply:
1. don't think that QAM_AUTOFS is right name. the fact that person from QAM created it is useless information from codebase perspective which is used by so many departments and even outside company ( openSUSE ). You can name it whatever you like but without department in name 
2. @mimi1vx  didn't get why usage of `QAM_AUTOFS` is blocking from using this in openSUSE ?

Reply:
Agreeing with both, please use a sensible variable name that you also document in variables.md and please test for openSUSE as well.

Our tests should always consider openSUSE Tumbleweed as the default product. Ensuring something working for openSUSE Tumbleweed will help all downstream products implicitly as well and should also be easier to test in all cases.

Reply:
@okurz excuse me, where should i insert my elsif (get_var('QAM_AUTOFS')) for opensuse.
products/opensuse/main.pm or lib/main_common.pm ?

Reply:
As you want to schedule the "autofs"-tests for both SLE and openSUSE then the schedule should go into lib/main_common.pm to not need to duplicate.

Reply:
the schedule now is in lib/main_common.pm . It has a dedicated function _load_mm_autofs_tests_ 
that one schedules only mm tests, local version is scheduled in _load_extra_tests_filesystem_
in lib/main_common

Reply:
@asmorodskyi it is just `AUTOFS` now
@mimi1vx it seems working now in opensuse